Pod/ILGClasses/ILGClasses.h

```diff
@@ -8,6 +8,8 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I think maybe it's a matter of perspective: Swift force-unwraps anything not explicitly annotated as nullable if you *don't* have nullability annotations set up. I feel this winds up being a shorter way to the same point: "Unless I tell you otherwise, this won't be null."